library/test/src/test_result.rs

```diff
@@ -105,6 +102,9 @@ pub fn get_result_from_exit_code(
             }
             None => unreachable!("status.code() returned None but status.signal() was None"),
         },
+        // Upon an abort, Fuchsia returns the status code ZX_TASK_RETCODE_EXCEPTION_KILL.
+        #[cfg(target_os = "fuchsia")]
+        Some(-1028) => TestResult::TrFailed,
```

Comment:
basically same question: why not `const ZX_TASK_RETCODE_EXCEPTION_KILL`?

Reply:
See response above. I added the constant you suggested and also added some documentation from some research I did on the origin of the code.